README.md

```diff
@@ -48,6 +48,8 @@ If the basic registers change you can restart your register so that it sees the
 - Install and run postgres db
 - Checkout project
 - Run the `./go` script to set everything up
+- `$ git submodule init`
+- `$ git submodule update`
```

Comment:
@danmitchell- Are you saying that the `git submodule update` should be run manually by a user and not as part of the `go` script in case they do not realise that the `go` script is doing this update?